### ISSUE TITLE
[OTHER] Question: is possible to send a new message and afterwards get the replies?

### ISSUE THREAD

**Issue by aislanspeedio** [linvo-io/linvo-scraper#22]
### What would you like to share?

I would like to be able to send messages to a new contact and asynchronously get the incoming replies from chat.

It is possible with this lib?

**Comment by nevo-david** [comment]
Yes, linkedin.message.service.ts, send a message and also scrape the inbox after
